tools/compass-installer/compass.Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG INSTALLER_VERSION="PR-8638"
```

Comment:
Maybe we could add a comment to this ARG, saying that this is the version of Kyma Operator (from Kyma repository) that will be used as a base for Compass Installer?